lib/elixir/lib/io.ex

```diff
@@ -171,14 +178,13 @@ defmodule IO do
   end
 
   @doc """
-  Writes the argument to the device, similar to `write/2`,
-  but adds a newline at the end. The argument is expected
-  to be a chardata.
+  Writes the given argument to the `device`, similar to `write/2`,
```

Comment:
sure... but i realized the origina version had too many "given"s
how about this?
> Writes `item` as a binary to the given `device`.